structopt-derive/src/attrs.rs

```diff
@@ -178,6 +180,9 @@ impl Attrs {
                 Word(ref w) if w == "flatten" => {
                     self.set_kind(Kind::FlattenStruct);
                 }
+                Word(ref w) if w == "long" => {
+                    self.is_autokebab = true;
```

Comment:
The field does not have a good name. Thats more something like auto_long_kebab.

Why not taking as name the kebab translation, and then long just doing self.add_str_method(self.name) ?